index.html

```diff
@@ -772,36 +773,1269 @@ <h2 id="BaseAudioContext">
+                <dfn><code id=
+                "idl-def-AudioContextState.suspended">suspended</code></dfn>
+              </td>
+              <td>
+                <p>
+                  This context is currently suspended (context time is not
+                  proceeding, audio hardware may be powered down/released).
+                </p>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <dfn><code id=
+                "idl-def-AudioContextState.running">running</code></dfn>
+              </td>
+              <td>
+                Audio is being processed.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <dfn><code id=
+                "idl-def-AudioContextState.closed">closed</code></dfn>
+              </td>
+              <td>
+                This context has been released, and can no longer be used to
+                process audio. All system audio resources have been released.
+                <span class="synchronous">Attempts to create new Nodes on the
+                AudioContext will throw <code>InvalidStateError</code></span>.
+                (AudioBuffers may still be created, through <a href=
+                "#widl-BaseAudioContext-createBuffer-AudioBuffer-unsigned-long-numberOfChannels-unsigned-long-length-float-sampleRate">
+                createBuffer</a>, <a href=
+                "#widl-BaseAudioContext-decodeAudioData-Promise-AudioBuffer--ArrayBuffer-audioData-DecodeSuccessCallback-successCallback-DecodeErrorCallback-errorCallback">
+                decodeAudioData</a>, or the <a>AudioBuffer</a> constructor.)
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <pre class="idl">
+callback DecodeErrorCallback = void (DOMException error);
+
+callback DecodeSuccessCallback = void (AudioBuffer decodedData);
+
+interface BaseAudioContext : EventTarget {
+    readonly        attribute AudioDestinationNode destination;
+    readonly        attribute float                sampleRate;
+    readonly        attribute double               currentTime;
+    readonly        attribute AudioListener        listener;
+    readonly        attribute AudioContextState    state;
+    Promise&lt;void&gt;          resume ();
+                    attribute EventHandler         onstatechange;
+    AudioBuffer            createBuffer (unsigned long numberOfChannels, unsigned long length, float sampleRate);
+    Promise&lt;AudioBuffer&gt;   decodeAudioData (ArrayBuffer audioData, optional DecodeSuccessCallback successCallback, optional DecodeErrorCallback errorCallback);
+    AudioBufferSourceNode  createBufferSource ();
+    ConstantSourceNode     createConstantSource ();
+    ScriptProcessorNode    createScriptProcessor (optional unsigned long bufferSize = 0
+              , optional unsigned long numberOfInputChannels = 2
+              , optional unsigned long numberOfOutputChannels = 2
+              );
```

Comment:
Only do this fancy formatting if you want to... ReSpec will rewrite it tho.

Reply:
Yeah, I just cut and pasted that from the original displayed spec without touching the result.  It does a really nice job of reformatting too.